packages/web-components/src/components/va-button-pair/va-button-pair.scss

```diff
@@ -28,17 +28,19 @@
 
 /* Original Component Styles. */
 
-:host([continue]:not([continue='false']):not([uswds])) {
-  flex-direction: row;
-}
-
 :host(:not([uswds])) va-button::part(button) {
   width: 280px;
 }
 
-:host([continue]:not([continue='false']):not([uswds])) va-button + va-button::part(button) {
-  margin-left: 12px;
-  margin-right: 0;
+@media (min-width: 481px) {
+  :host([continue]:not([continue='false']):not([uswds])) {
+    flex-direction: row;
+  }
+
+  :host([continue]:not([continue='false']):not([uswds])) va-button + va-button::part(button) {
+    margin-left: 12px;
+    margin-right: 0;
+  }
```

Comment:
You will probably want to add `flex-direction: column-reverse;` like what we did in the v3 version in order to get the "continue" button on the top:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-button-pair/va-button-pair.scss#L24-L26